host_guest/Analysis/Submissions/README.md

```diff
@@ -3,3 +3,7 @@
 ## Manifest
 
 - `Isaacs_clip`: Submissions for the TrimerTrip/Isaacs lab component of the challenge.
+- `CD_REF.txt`: Reference calculations methods for Cyclodextrins/Gibbs lab host-guest systems
```

Comment:
One of these descriptions says "methods", the rest don't. I think you want to remove the word "methods".

You probably also want to update the overall file (not manifest) to explain who did these calculations, that they are not formal submissions, etc.